transformation/transform_json.py

```diff
@@ -48,6 +48,7 @@ class QueryCategory(Enum):
     AGGREGATE = "aggregate"
     ANALYTICAL = "analytical"
     UPDATE = "update"
+    BIG = "big"
```

Comment:
I'm not sure just about this Category; this should not exist.  The rest looks good! 🚀